spec/lightstep_spec.rb

```diff
@@ -111,6 +111,25 @@ def init_callback_tracer(callback)
     span.finish
   end
 
+  it 'should coerce all tag values into strings' do
+    class SampleClass
```

Comment:
This can be a single line declaration by replacing the newline with a semicolon, which is common in ruby.